homeassistant/components/history/__init__.py

```diff
@@ -327,6 +331,7 @@ def __init__(self, filters, use_include_order):
         if entity_ids:
             entity_ids = entity_ids.lower().split(",")
         include_start_time_state = "skip_initial_state" not in request.query
+        significant_changes_only = "significant_changes_only" in request.query
```

Comment:
This is a breaking change for anyone that is using this API because it used to default to True. Instead of checking presence in the query string, let's see if the value is 1 or 0.

```suggestion
        significant_changes_only = request.query.get("significant_changes_only", "1") != "0"
```

Reply:
agreed